src/client.js

```diff
@@ -0,0 +1,60 @@
+const axios = require('axios')
+const toNumber = require('lodash/toNumber')
+const UdpServer = require('./udp/index.js')
+
+const SERVER_URL = process.env.SERVER_URL || 'http://localhost:3000'
+const CLIENT_UDP_PORT = toNumber(process.env.CLIENT_UDP_PORT) || 41234
+const CLIENT_IP_ADDRESS = process.env.CLIENT_IP_ADDRESS || 'localhost'
+const udpClient = new UdpServer('upd-client')
+
+async function getServerInformation() {
+    try {
+        const response = await axios.get(`${SERVER_URL}/address`)
+        serverAddress = response.data
+    } catch (error) {
+        console.log('Something went wrong when getting server address: ', error.message);
+        serverAddress = null
+    }
+
+    return serverAddress
+}
+
+async function startConnection() {
+    const serverAddress = await getServerInformation()
+    if(!serverAddress) {
+        console.log('Cannot get udp server address. Exit!');
+        return
+    }
+    
+    console.log('Received udp server address: ', serverAddress);
+    const getServerInfoUrl = encodeURI(`${SERVER_URL}/client?address=${CLIENT_IP_ADDRESS}&port=${CLIENT_UDP_PORT}`)
```

Comment:
No need `CLIENT_IP_ADDRESS`, the client call https://www.ipify.org/ to get the outbound IP and send in req